src/rpc/net.cpp

```diff
@@ -587,7 +601,7 @@ static RPCHelpMan getnetworkinfo()
                         {
                             {RPCResult::Type::OBJ, "", "",
                             {
-                                {RPCResult::Type::STR, "name", "network (ipv4, ipv6 or onion)"},
+                                {RPCResult::Type::STR, "name", "network (" + Join(GetNetworkNames(), ", ") + ")"},
```

Comment:
I don't think getnetworkinfo returns an object for unroutable networks? Also, i2p, etc aren't implemented yet.

Reply:
Good catch about getnetworkinfo! (and I should have remembered that, too). Fixed both.